### PR TITLE
test(evm): Add tests for feemarket wrapper

### PR DESCRIPTION
**Pull request by 0xstepit**
# Description

This PR add tests for the feemarket wrapper used in the x/evm module to perform custom decimals conversions.

THIS BRANCH HAS BEEN CREATED OUT OF https://github.com/evmos/os/pull/65

## Release Notes

- **New Features**
  - Added comprehensive test coverage for FeeMarket wrapper in EVM module

- **Bug Fixes**
  - Improved error handling in `GetParams` method by adding nil check for BaseFee parameter

- **Tests**
  - Introduced unit tests for `GetBaseFee`, `CalculateBaseFee`, and `GetParams` methods
  - Validated wrapper functionality under various scenarios including decimal conversions and edge cases

### CODE REVIEW

**Review by GAtom22 — APPROVED** [review]
LGTM! Thanks @0xstepit!!

**Review by hanchon — APPROVED** [review]

File: x/evm/wrappers/feemarket_test.go [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+			mockFeeMarketKeeper := testutil.NewMockFeeMarketKeeper(ctrl)
+			tc.mockSetup(mockFeeMarketKeeper)
+
+			feeMarketWrapper := wrappers.NewFeeMarketWrapper(mockFeeMarketKeeper)
+			result := feeMarketWrapper.GetParams(sdk.Context{})
+
+			require.Equal(t, tc.expParams, result)
+		})
+	}
+}
```
[the remark this answers is not in the document]
Comment (0xstepit) [reply x/evm/wrappers/feemarket_test.go:269]: This test is not intended to check the correctness of the feemarket params but only the proper conversion of the base fee and min gas price
